File: cash.py
import cs50

#Get number
while True:
    cash = cs50.get_float("Change owed: ")
    if (cash > 0):
        break

cash = cash * 100
count = 0

#Count coins
while cash >= 25:
    cash = cash - 25
    count += 1

while cash >= 10:
    cash = cash - 10
    count += 1

while cash >= 5:
    cash = cash - 5
    count += 1
    
while cash >= 1:
    cash = cash- 1
    count += 1
  
#Output count 
print(count)

File: cash.c
//Libraries
#include <cs50.h>
#include <math.h>
#include <stdio.h>

//Variables
float dollars;
void get_number();
int cents;
int count;

//Main program
int main()
{
    //Get positive number
    get_number();
    //Convert dollars to cents
    cents = round(dollars * 100);
    //Calculate how many coins
    //
    //25 cents
    while (cents >= 25)
    {
        cents = (cents - 25);
        count = (count + 1);
    }
    //10 cents
    while (cents >= 10)
    {
        cents = (cents - 10);
        count = (count + 1);
    }
    //5 cents
    while (cents >= 5)
    {
        cents = (cents - 5);
        count = (count + 1);
    }
    //1 cent
    while (cents >= 1)
    {
        cents = (cents - 1);
        count = (count + 1);
    }


    //Print coins
    printf("%i\n", count);
    
}





//Get positive number
void get_number()
{
    do 
    {
        dollars = get_float("Amount of dollars: $");
    }
    while (dollars < 0);
}